js/app.js

```diff
@@ -243,10 +243,10 @@ OWI.config(['$stateProvider', '$urlRouterProvider', function($stateProvider, $ur
 // Listen for state change errors such as routing to an invalid hero and redirect
 OWI.run(["$transitions", "$state", "DataService", function($transitions, $state, Data) {
   $transitions.onError({}, function(error) {
-    if (error.error().detail === 'INVALID_HERO') {
-      $state.go('home');
-    } else {
+    if (error.error().detail === 'INVALID_EVENT') {
       $state.go('events', { id: Data.currentEvent });
+    } else if (error.error().type !== RejectType.IGNORED) {
```

Comment:
`RejectType` doesn't exist. You should just get rid of the whole else if here and just make it an else.

Reply:
Oh right i forgot the reason for initially doing it, where does RejectType come from?

Reply:
Yeah, I noticed this and I'm trying to figure out how to access the enum right now.
If there isn't a way, [`RejectType.IGNORED`](https://ui-router.github.io/ng1/docs/latest/enums/transition.rejecttype.html#ignored) can just be replaced by it's value, 5.
It's necessary to stop the infinite loop if the ignored transition is the same page, i.e. clicking the home button on the home page.

Reply:
Yeah I was just gonna say, just check for the error code 5 which seems to be the ignored code.

Reply:
Ok, I'll commit that change right now if you want to merge, and maybe look into a better way later.